modules/helm/format.go

```diff
@@ -0,0 +1,81 @@
+package helm
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/gruntwork-cli/collections"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/require"
+)
+
+// FormatSetValuesAsArgs formats the given values as command line args for helm using set (e.g of the format
+// --set key=value)
+func FormatSetValuesAsArgs(setValues map[string]string) []string {
+	args := []string{}
+
+	// To make it easier to test, go through the keys in sorted order
+	keys := collections.Keys(setValues)
+	for _, key := range keys {
+		value := setValues[key]
+		argValue := fmt.Sprintf("%s=%s", key, value)
+		args = append(args, "--set", argValue)
+	}
+
+	return args
+}
+
+// FormatValuesFilesAsArgs formats the given list of values file paths as command line args for helm (e.g of the format
+// -f path). This will fail the test if one of the paths do not exist.
+func FormatValuesFilesAsArgs(t *testing.T, valuesFiles []string) []string {
+	args, err := FormatValuesFilesAsArgsE(t, valuesFiles)
+	require.NoError(t, err)
+	return args
+}
+
+// FormatValuesFilesAsArgsE formats the given list of values file paths as command line args for helm (e.g of the format
+// -f path)
+func FormatValuesFilesAsArgsE(t *testing.T, valuesFiles []string) ([]string, error) {
+	args := []string{}
+
+	for _, valuesFilePath := range valuesFiles {
+		// Pass through filepath.Abs to make sure this file exists
+		absValuesFilePath, err := filepath.Abs(valuesFilePath)
```

Comment:
I don't think `Abs` checks for existence of the file...

Reply:
Ah you are right! Fixed to explicitly check for existence

Reply:
Make sure to update this in all the places you use `Abs`!

Reply:
Did a quick grep and I think I was the only idiot who thought this.